requirements.txt

```diff
@@ -19,4 +19,10 @@ albumentations>=1.3.0
 pytube>=12.1.3
 pyffmpeg==2.0
 GitPython>=3.1.18
+pytorch-metric-learning==1.6.3
+faiss-cpu==1.7.4
```

Comment:
Would using faiss-gpu make any difference?

Reply:
It wouldn't, because we are using the CustomKNN solution from the pml library instead of faiss-based solutions. However, AccuracyCalculator function from the pml, imports FaissKNN in every scenario and uses it if there is no CustomKNN defined. Therefore, this is added to prevent import errors.